packages/grafana-data/src/dataframe/processDataFrame.ts

```diff
@@ -449,11 +447,11 @@ export function reverseDataFrame(data: DataFrame): DataFrame {
   return {
     ...data,
     fields: data.fields.map((f) => {
-      const copy = [...f.values.toArray()];
-      copy.reverse();
+      const values = [...f.values];
+      values.reverse();
```

Comment:
probably better/faster as `values: f.values.slice().reverse()`. but we'll leave that for future.